mscore/inspector/inspector_ambitus.ui

```diff
@@ -846,8 +846,11 @@
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
+        <property name="toolTip">
+         <string>Adjust the range of the ambitus to match the range of the notes that are currently on the staff.</string>
```

Comment:
"...currentl **used** on..."?

Reply:
I cannot say myself. I don't feel like the "_used_" is necessary.
I'll add it if someone agrees with you.